site/content/docs/manifest/worker-service.en.md

```diff
@@ -216,6 +241,9 @@ The acceptable amount of time that a message can sit in the queue. For example,
 <span class="parent-field">count.queue_delay.</span><a id="count-queue-delay-msg-processing-time" href="#count-queue-delay-msg-processing-time" class="field">`msg_processing_time`</a> <span class="type">Duration</span>   
 The average amount of time it takes to process an SQS message. For example, `"250ms"`, `"1s"`.
 
+<span class="parent-field">count.queue_delay.</span><a id="count-queue-delay-cooldown" href="#count-queue-delay-cooldown" class="field">`cooldown`</a> <span class="type">Map</span>  
```

Comment:
I did it this way because `queue_delay` isn't the same `ScalingConfigOrT[]` type as the rest of the scaling fields, and so I explicity defined the cooldown for `queue_delay`, and then defined the cooldown for the others in the parent field in the same format as how they were specified in the `backend-service` and `lb-web-service`.